google/ads/google_ads/config.py

```diff
@@ -13,22 +13,36 @@
 # limitations under the License.
 """A set of functions to help load configuration from various locations."""
 
-import json
 import functools
+import json
+import logging.config
 import os
 import yaml
 
+
+_logger = logging.getLogger(__name__)
+
 _ENV_PREFIX = "GOOGLE_ADS_"
 _REQUIRED_KEYS = ("developer_token",)
-_OPTIONAL_KEYS = ("login_customer_id", "endpoint", "logging")
+_OPTIONAL_KEYS = (
+    "login_customer_id",
+    "endpoint",
+    "logging",
+    "configuration_file_path",
+    "linked_customer_id",
+)
 _OAUTH2_INSTALLED_APP_KEYS = ("client_id", "client_secret", "refresh_token")
 _OAUTH2_SERVICE_ACCOUNT_KEYS = ("path_to_private_key_file", "delegated_account")
+# These keys are additional environment variables that can be used to specify
+# some of the above configuration values.
+_REDUNDANT_KEYS = ("json_key_file_path", "impersonated_email")
```

Comment:
I changed the names here so that the new keys are the primary and the old ones are named "secondary." Additionally the new names are used throughout the rest of the codebase so that we can more easily remove the secondary ones later.

Reply:
Thanks!